src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -130,6 +130,7 @@ public static class ScriptConstants
         public const string FeatureFlagRelaxedAssemblyUnification = "RelaxedAssemblyUnification";
         public const string FeatureFlagEnableDiagnosticEventLogging = "EnableDiagnosticEventLogging";
         public const string FeatureFlagDisableDiagnosticEventLogging = "DisableDiagnosticEventLogging";
+        public const string FeatureFlagDisableWeHostLogForwarding = "FlagDisableWeHostLogForwarding";
```

Comment:
spelling:
```suggestion
        public const string FeatureFlagDisableWebHostLogForwarding = "FlagDisableWeHostLogForwarding";
```

Reply:
I have removed Flag from the value to maintain consistency with other feature flags.